test/e2e/nsx_vpc_test.go

```diff
@@ -0,0 +1,143 @@
+package e2e
+
+import (
+	"log"
+	"path/filepath"
+	"strings"
+	"testing"
+)
+
+const (
+	VPCCRName             = "vpcs.nsx.vmware"
+	VPCCRType             = "vpcs"
```

Comment:
I'd still suggest to separate VPC cr type with VPC nsx type, although somehow it could work on nsx, but we do not know what will change in the future, and we do not need to simplify here by just saving one parameter which may leave us potential risk in the future.